test/functional/cypress/specs/companies/lead-advisers/manage-adviser-spec.js

```diff
@@ -1,121 +1,103 @@
 const { assertBreadcrumbs } = require('../../../support/assertions')
 const urls = require('../../../../../../src/lib/urls')
-const selectors = require('../../../../../selectors/index')
 
 const EXPECTED_NAME = 'Andy Pipkin'
 const EXPECTED_TEAM = 'Andy & Lou'
 
-const companyLocalHeader = selectors.companyLocalHeader()
-
-const selectMainContent = () =>
-  cy
-    .get('main')
-    .children()
-    .first()
-
 const addOrReplaceTestCase = ({
   companyId,
   companyName,
   name,
   team,
-  replace,
+  replace = false,
 }) =>
-  it(`Should render the ${replace ? 'replace' : 'add'} page`, () => {
+  it(`Should render "${replace ? 'replace' : 'add'} a lead adviser" ${
+    !team && replace ? 'with no team' : ''
+  }`, () => {
     cy.visit(urls.companies.advisers.assign(companyId))
 
     const headline = replace
       ? 'Replace the Lead ITA'
-      : 'Confirm you are the Lead ITA'
-
-    assertBreadcrumbs({
-      Home: urls.dashboard(),
-      Companies: urls.companies.index(),
-      [companyName]: urls.companies.detail(companyId),
-      [headline]: undefined,
-    })
+      : 'Add someone as the Lead ITA'
+
+    replace
+      ? assertBreadcrumbs({
+          Home: urls.dashboard(),
+          Companies: urls.companies.index(),
+          [companyName]: urls.companies.detail(companyId),
+          'Replace the Lead ITA': undefined,
+        })
+      : assertBreadcrumbs({
+          Home: urls.dashboard(),
+          Companies: urls.companies.index(),
+          [companyName]: urls.companies.detail(companyId),
+          'Add Lead ITA': undefined,
+        })
```

Comment:
There is no need to copy the entire block

```suggestion
    assertBreadcrumbs({
      Home: urls.dashboard(),
      Companies: urls.companies.index(),
      [companyName]: urls.companies.detail(companyId),
      [replace ?  'Replace the Lead ITA' :  'Add Lead ITA']: undefined,
    })
```